endpoints/openrtb2/sample-requests/valid-whole/supplementary/site-stored-response.json

```diff
@@ -1,80 +0,0 @@
-{
```

Comment:
I can definitely bring this file back. I discarded it in the last commit because I thought it was too similar to `endpoints/openrtb2/sample-requests/valid-whole/supplementary/imp-with-stored-resp.json`. Do you think they differ enough to bring it back?

Reply:
Discussed offline. There's no need to bring this test back. AMP stored responses are adequately covered by the `imp-with-stored-resp.json` test.